packages/react/src/alert-dialog/popup/AlertDialogPopup.tsx

```diff
@@ -53,22 +52,11 @@ export const AlertDialogPopup = React.forwardRef(function AlertDialogPopup(
     titleElementId,
     transitionStatus,
     modal,
-    onOpenChangeComplete,
     internalBackdropRef,
   } = useAlertDialogRootContext();
 
   useAlertDialogPortalContext();
 
-  useOpenChangeComplete({
-    open,
-    ref: popupRef,
-    onComplete() {
-      if (open) {
-        onOpenChangeComplete?.(true);
-      }
-    },
-  });
```

Comment:
This can't be moved from this component. The root is unable to handle the `open: true` case (it just ends up firing it instantly instead of waiting for the animation to finish)